src/libstd/keyword_docs.rs

```diff
@@ -166,15 +183,15 @@ mod crate_keyword { }
 ///
 /// For more information, take a look at the [Rust Book] or the [Reference]
 ///
-/// [Algebraic Data Types]: https://en.wikipedia.org/wiki/Algebraic_data_type
+/// [ADT]: https://en.wikipedia.org/wiki/Algebraic_data_type
 /// [`Option`]: option/enum.Option.html
 /// [Rust Book]: ../book/ch06-01-defining-an-enum.html
 /// [Reference]: ../reference/items/enumerations.html
 mod enum_keyword { }
 
 #[doc(keyword = "extern")]
 //
-/// For external connections in Rust code.
+/// Link to or import external code.
```

Comment:
extern is used for specifying the ABI. So "link to or import" is not really true.

Reply:
"import external code" alludes to `extern crate foo` syntax, which is still valid but pretty much useless on 2018, maybe this should be removed?

Reply:
Both should be kept.